mods/ts/rules/defaults.yaml

```diff
@@ -357,6 +357,16 @@
 	TimedUpgradeBar@EMPDISABLE:
 		Upgrade: empdisable
 		Color: 255,255,255
+	WithDecoration@EMPDISABLE:
+		Image: emp
+		Sequence: idle
+		ReferencePoint: HCenter, VCenter
+		Palette: effect
+		UpgradeTypes: empdisable
+		ShowToEnemies: true
+		ZOffset: 8192
```

Comment:
This is rather bogus.  Can we fix it properly, or please add a # TODO: FIXME comment.